kolibri/content/hooks.py

```diff
@@ -0,0 +1,78 @@
+"""
+Kolibri Content hooks
+---------------------
+
+Hooks for managing the display and rendering of content.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import json
+import logging
+import os
+
+from django.conf import settings as django_settings
+from django.utils.functional import cached_property
+from django.utils.safestring import mark_safe
+from kolibri.core.webpack.hooks import WebpackBundleHook
+from le_utils.constants import content_kinds
+
+logger = logging.getLogger(__name__)
+
+class ContentRendererHook(WebpackBundleHook):
+    """
+    An inheritable hook that allows special behaviour for a frontend module that defines
+    a content renderer.
+
+    Reads a JSON file of this format:
+    {
+        "kind": [
+            "file_extension"
+        ]
+    }
+    e.g.
+    {
+        "video": [
+            "mp4",
+            "ogg",
+            "wmv"
+        ]
+    }
```

Comment:
Sounds like a good plan to me, a few characters is a small price to pay for user friendly JSON!